src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagService.cs

```diff
@@ -48,18 +56,31 @@ public async Task<AddCustomTagResponse> AddCustomTagAsync(IEnumerable<CustomTagE
             _customTagEntryValidator.ValidateCustomTags(customTags);
 
             Dictionary<long, CustomTagStoreEntry> addedTags = new Dictionary<long, CustomTagStoreEntry>();
-            foreach (var tag in customTags)
+            foreach (var customTag in customTags)
             {
                 try
                 {
                     CustomTagStatus initStatus = CustomTagStatus.Reindexing;
-                    long key = await _customTagStore.AddCustomTagAsync(tag.Path, tag.VR, tag.Level, initStatus);
-                    CustomTagStoreEntry storeEntry = new CustomTagStoreEntry(key, tag.Path, tag.VR, tag.Level, initStatus);
+
+                    // we only support path that has 1 attrubteId for now, will support path which is composed of multiple attributeId later (e.g: 00100010.00100020)
```

Comment:
we only support top level dicomtag right now, will support embeded dicomtag soon